zookeeper-server/src/main/java/org/apache/zookeeper/server/FinalRequestProcessor.java

```diff
@@ -565,6 +566,12 @@ public void processRequest(Request request) {
                 rsp = new GetEphemeralsResponse(ephemerals);
                 break;
             }
+            case OpCode.whoAmI: {
+                lastOp = "HOMI";
+                rsp = new WhoAmIResponse(request.getUsers(true));
+                break;
+            }
+
```

Comment:
This is where the builds break; it wants a `default:` in this switch:

> `[ERROR] Switch statement found in org.apache.zookeeper.server.FinalRequestProcessor.processRequest(Request) where default case is missing [org.apache.zookeeper.server.FinalRequestProcessor] At FinalRequestProcessor.java:[lines 212-571] SF_SWITCH_NO_DEFAULT`

(Not something you introduced, AFAICT, but "you touch it, you own it" seems to be the policy :)

Reply:
Handled